microprofile-health/src/test/java/org/jboss/eap/qe/microprofile/health/integration/FailSafeCDIHealthBaseTest.java

```diff
@@ -57,17 +72,29 @@ public void before() throws ConfigurationException, InterruptedException, Timeou
         new Administration(ManagementClientProvider.onlineStandalone(managementClient)).reload();
```

Comment:
See my comment above, about evaluating `@BeforeClass` for this one too.

Reply:
This also resets MP Metrics...will change the comment. This must be here.

Reply:
I see, so that is meant to reset Metrics _before_each test, right? And this applies to the above comment as well, I guess. 
I am asking because I was concerned about the order of execution for `@Before` methods in the hierarchy (see https://junit.org/junit4/javadoc/latest/org/junit/Before.html - superclass comes _first_) so this is good if the goal is _just_ to reset metrics.
The metrics URL could be computed just once here too, IMO, WDYT?

Reply:
You are right, I must have missed this one.